libmscore/scale.h

```diff
@@ -0,0 +1,50 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2002-2012 Werner Schweer
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __SCALE_H__
+#define __SCALE_H__
+
+/**
+ \file
+ Definition of Scale class.
+ */
+
+#include "note.h"
+
+namespace Ms {
+
+class Scale {
+public:
+      bool loadScale(const QString&);
+      double getTuning(const Note* note) const;
+
+private:
+      QString getNextLine(QTextStream& in);
+
+      std::vector<double> originalNotes;
+      std::map<Tpc, double> computedNotes;
+
```

Comment:
Also, if I am not mistaken, do we really need fractionary cents? Wouldn't rounding to the nearest `int` enough? I doubt anyone can notice a difference of 0.5 cent.